dist/rename.py

```diff
@@ -66,14 +72,127 @@ def get_episode_from_media(filename: str, seasons: dict[str, int]) -> Optional[E
             show=SHOW_NAME,
             season=season_number,
             number=episode_number,
+            extended=match.group(4) is not None,
+            filepath=filepath,
         )
 
 
 def debugger_is_active() -> bool:
     return hasattr(sys, "gettrace") and sys.gettrace() is not None
 
 
+def ensure_tag_value(root, tag, value):
+    if root.find(tag) is None:
+        el = ET.Element(tag)
+        el.text = value
+        root.append(el)
+        return True
+    if root.find(tag).text != value:
+        root.find(tag).text = value
+        return True
+    return False
+
+
+def fix_season_nfo(fpath: Path, sno: int, sname: str):
+    try:
+        tree = ET.parse(str(fpath.absolute()))
+    except Exception as e:
+        print(fpath, e)
+        return
+    root = tree.getroot()
+    to_remove = []
+    for child in root:
```

Comment:
It may be simpler to do `for child not in root` and explicitly state the tags we want. If TinyMediaManager ever wrote an unneeded tag that isn't listed here it would be missed

Reply:
that makes lot of sense, just adjusted it (and reduced duplication)